<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Game_Setup</class>
 <widget class="QDialog" name="Game_Setup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>288</width>
    <height>166</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Game setup</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QGroupBox::title { 
	subcontrol-origin: padding; 
	subcontrol-position: top center;
	margin-top: 0.25em;
	font-style: italic;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="Bloqus::Gui::Player_Setup" name="north_west_setup">
       <property name="title">
        <string notr="true">north_west</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="Bloqus::Gui::Player_Setup" name="north_east_setup">
       <property name="title">
        <string notr="true">north_east</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="Bloqus::Gui::Player_Setup" name="south_west_setup">
       <property name="title">
        <string notr="true">south_west</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="Bloqus::Gui::Player_Setup" name="south_east_setup">
       <property name="title">
        <string notr="true">south_east</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Allowed time per player (minutes):</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QSlider" name="allowed_time">
       <property name="minimum">
        <number>4</number>
       </property>
       <property name="maximum">
        <number>30</number>
       </property>
       <property name="pageStep">
        <number>1</number>
       </property>
       <property name="value">
        <number>10</number>
       </property>
       <property name="sliderPosition">
        <number>10</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksBothSides</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLCDNumber" name="lcd_time">
       <property name="smallDecimalPoint">
        <bool>false</bool>
       </property>
       <property name="numDigits">
        <number>5</number>
       </property>
       <property name="segmentStyle">
        <enum>QLCDNumber::Flat</enum>
       </property>
       <property name="intValue" stdset="0">
        <number>5</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="bt_cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="bt_play">
       <property name="text">
        <string>Play!</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Bloqus::Gui::Player_Setup</class>
   <extends>QGroupBox</extends>
   <header>player_setup.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>bt_cancel</sender>
   <signal>clicked()</signal>
   <receiver>Game_Setup</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>59</x>
     <y>154</y>
    </hint>
    <hint type="destinationlabel">
     <x>0</x>
     <y>165</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bt_play</sender>
   <signal>clicked()</signal>
   <receiver>Game_Setup</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>277</x>
     <y>155</y>
    </hint>
    <hint type="destinationlabel">
     <x>205</x>
     <y>165</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
